src/models/AudioStorage.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from "mongoose";
+
+const AudioStorageSchema = new mongoose.Schema(
+  {
+    audioContent: Buffer,
+    audioURL: String,
+    audioType: String,
+    audioLength: mongoose.Decimal128,
+  },
+  {
+    timestamps: true,
```

Comment:
we no ned to knw the timestamp actually,
if really need will only required createdAt of the storage data maybe
because the media(audio,image, video) storage will not be update